tcms/templates/run/report.html

```diff
@@ -143,14 +143,6 @@ <h4 style='padding:0px;margin:0px'>Comments</h4>
 				<td><a href="{{ bug_url }}" target="_blank">{{ bug_url }}</a></td>
 			</tr>
 			{% endfor %}
-			<tr>
-				<td>View all bugs(Bugzilla)</td>
-				<td><a href="{{ bugzilla_url }}" target="_blank">{{ bugzilla_url }}</a></td>
-			</tr>
-			<tr>
-				<td>View all bugs(JIRA)</td>
-				<td><a href="{{ jira_url }}" target="_blank">{{ jira_url }}</a></td>
-		  </tr>
```

Comment:
Well, this commit aims to use bug trackers defined in database, then all hardcoded bug tracker settings should be read from database.

So, both ``bugzilla_url`` and ``jira_url`` should also be defined in and read from db along with ``Check to add Test Cases to BZ``.